project.php

```diff
@@ -0,0 +1,21 @@
+<?php
+header('Content-Type: application/json; charset=utf-8');
+header("Access-Control-Allow-Origin: *");
+require 'vendor/autoload.php';
+
+try {
+    $db = \ProjectManager\Services\DbConnector::connect();
+        if (isset($_GET['id']) && is_numeric($_GET['id'])) {
```

Comment:
why is_numeric instead of is_int?

Reply:
Changing to is_int it seems that the $_GET variable doesn't read the Id and goes straight to the error with Invalid_ID message